src/modules/notifications/components/NotificationPanel.js

```diff
@@ -0,0 +1,51 @@
+import { html } from 'lit-html';
+import { $injector } from '../../../injection';
+import css from './notificationPanel.css';
+import { AbstractContentPanel } from '../../menu/components/mainMenu/content/AbstractContentPanel';
+import { observe } from '../../../utils/storeUtils';
+import { NotificationItem } from './NotificationItem';
+
+
+/**
+ * Container for notifications.
+ * @class
+ * @author thiloSchlemmer
+ */
+export class NotificationPanel extends AbstractContentPanel {
+
+
+	constructor() {
+		super();
+		const { TranslationService } = $injector.inject('TranslationService', 'GeoResourceService');
+		this._translationService = TranslationService;
+		this._registeredObservers = this._register(this._storeService.getStore());
+	}
+
+	/**
+	 * @override
+	 */
+	createView() {
+		return html`
+        <style>${css}</style>
+		<div class="notification-panel">
+		</div>
+        `;
+	}
+
+	_showNotification(e) {
+		const parent = this.shadowRoot.querySelector('.notification-panel');
+		const item = document.createElement(NotificationItem.tag);
+		item.content = { notification: e.payload, autocloseTime: 10000 };
+		parent.appendChild(item);
+	}
+
```

Comment:
Works, but does not make use of the BaElement philosophy of state handling...
See: 
- https://github.com/ldbv-by/bav4-nomigration/tree/dev_guide/src/modules/guide/components
- https://github.com/ldbv-by/bav4-nomigration/blob/dev_guide/src/modules/guide/components/GlobalState.js